app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +13,15 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        //
+        $this->app->bind(
+            \CyrildeWit\EloquentViewable\Contracts\View::class,
+            \App\Models\View::class
```

Comment:
It doesn't look like this view file exists.

Reply:
Ahh, should be good now.